data/tlg0341/tlg002/tlg0341.tlg002.perseus-grc1.xml

```diff
@@ -1,1606 +1,1604 @@
-<l n="1454">πίστιν γὰρ ἡμῶν Λεψιεὺς ἐνόσφισε,</l>
-<l n="1455">ψευδηγόροις φήμαισιν ἐγχρίσας ἔπη, </l>
-<l n="1456">καὶ θεσφάτων πρόμαντιν ἀψευδῆ φρόνιν,</l>
-<l n="1457">λέκτρων στερηθεὶς ὧν ἐκάλχαινεν τυχεῖν.</l>
-<l n="1458">θήσει δʼ ἀληθῆ. σὺν κακῷ δέ τις μαθών,</l>
-<l n="1459">ὅτʼ οὐδὲν ἔσται μῆχος ὠφελεῖν πάτραν,</l>
-<l n="1460">τὴν φοιβόληπτον αἰνέσει χελιδόνα. </l>
-<l n="1461">Τόσσʼ ἠγόρευε, καὶ παλίσσυτος ποσὶν</l>
-<l n="1462">ἔβαινεν εἱρκτῆς ἐντός. ἐν δὲ καρδίᾳ</l>
-<l n="1463">σειρῆνος ἐστέναξε λοίσθιον μέλος,</l>
-<l n="1464">Κλάρου Μιμαλλών, ἢ Μελαγκραίρας κόπις</l>
-<l n="1465">Νησοῦς θυγατρός, ἤ τι Φίκιον τέρας, </l>
-<l n="1466">ἑλικτὰ κωτίλλουσα δυσφράστως ἔπη.</l>
-<l n="1467">ἐγὼ δὲ λοξὸν ἦλθον ἀγγέλλων, ἄναξ,</l>
-<pb id="p.616"/>
-<l n="1468">σοὶ τόνδε μῦθον παρθένου φοιβαστρίας,</l>
-<l n="1469">ἐπεί μʼ ἔταξας φύλακα λαΐνου στέγης</l>
-<l n="1470">καὶ πάντα φράζειν κἀναπεμπάζειν λόγον </l>
-<l n="1471">ἐτητύμως ἄψορρον ὤτρυνας τρόχιν.</l>
-<l n="1472">δαίμων δέ φήμας εἰς τὸ λῷον ἐκδραμεῖν</l>
-<l n="1473">τεύξειεν, ὅσπερ σῶν προκήδεται θρόνων,</l>
-<l n="1474">σώζων παλαιὰν Βεβρύκων παγκληρίαν.</l>
-</div1>
-</body>
-</text>
-</TEI.2>
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="http://www.stoa.org/epidoc/schema/9.1/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:lang="en" xmlns:tei="http://www.tei-c.org/ns/1.0">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="grc">Ἀλεξάνδρα</title>
+                <author>Lycophron</author>
+                <sponsor>Perseus Project, Tufts University</sponsor>
+                <principal>Gregory Crane</principal>
+                <respStmt>
+                    <resp>Prepared under the supervision of</resp>
+                    <name>Lisa Cerrato</name>
+                    <name>William Merrill</name>
+                    <name>Elli Mylonas</name>
+                    <name>David Smith</name>
+                </respStmt>
+                <funder n="org:Google">Google Digital Humanities Awards Program</funder>
+            </titleStmt>
+            <extent />
+            <publicationStmt>
+                <publisher>Trustees of Tufts University</publisher>
+                <pubPlace>Medford, MA</pubPlace>
+                <authority>Perseus Project</authority>
+            </publicationStmt>
+            <sourceDesc>
+                <biblStruct>
+                    <monogr>
+                        <author>Lycophron</author>
+                        <title>Alexandra</title>
+                        <editor>A.W. Mair</editor>
+                        <imprint>
+                            <publisher>London: William Heinemann; New York: G.P. Putnam's Sons</publisher>
+                            <date>1921</date>
+                        </imprint>
+                    </monogr>
+                </biblStruct>
```

Comment:
need ref target for pdf source

example:
 `<ref target="https://archive.org/details/falloftroywithen00quinuoft/page/2/mode/2up">The Internet Archive</ref>`

Reply:
Source should match the edition and be of good (legible) quality. Internet Archive preferred over Google.

https://catalog.perseus.org/catalog/urn:cts:greekLit:tlg0341.tlg002.perseus-grc1

Reply:
We can strike the `?view=theater` here